agagd/templates/base.beta.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+
+{%  load static %}
+
+<html> 
+  <head>
+    <title>{% block title %} AGAGD | American Go Game Database {% endblock title %}</title>
+        {% block css %}
+            <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css" integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l" crossorigin="anonymous">
+        {% endblock %}
+
+        {% block js %}
+            <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
```

Comment:
What's `jquery` for?

Reply:
I suspect it will be unused. I don't think the legacy uses it either other than for possibly the tournament section's date.

Reply:
We seem not to be using any of the features of Bootstrap's JavaScript, so I have removed both the jquery's js and bootstrap's js.